<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QConducao</class>
 <widget class="QMainWindow" name="QConducao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QConducao</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QLineEdit" name="InputDx">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="LabelDxUnit">
      <property name="text">
       <string>m</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="LabelFluxUnit">
      <property name="text">
       <string>W</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="InputDt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QTableWidget" name="TabelaDados">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Temperatura</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Condutividade</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Temperatura</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="LabelCondutUnit">
      <property name="text">
       <string>W/°C.m²</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QPushButton" name="BtnCalc">
      <property name="text">
       <string>Calcular</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="BoxMaterial"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>dx</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Fluxo Calor</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="LabelDtUnit">
      <property name="text">
       <string>°C</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>dt</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="InputArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="InputCondut">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="LabelTempUnit">
      <property name="text">
       <string>°C</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="BtnAddPonto">
        <property name="text">
         <string>Adicionar Ponto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnRemovePonto">
        <property name="text">
         <string>Remover Ponto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCriaCurva">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Criar Cruva</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnLimpa">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="InputTemp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Condutividade</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="InputFlux">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Área</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>m²</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <widget class="QTabWidget" name="AbasGrafico">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="AbaMaterial">
       <attribute name="title">
        <string>Gráfico Material</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="Plot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AbaGeral">
       <attribute name="title">
        <string>Gráfico Geral</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="PlotGeral" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="MenuNovoMaterial"/>
    <addaction name="MenuCarregar"/>
    <addaction name="MenuSalvarTabela"/>
    <addaction name="MenuSalvarEquacoes"/>
    <addaction name="MenuSair"/>
   </widget>
   <widget class="QMenu" name="menuGrafico">
    <property name="title">
     <string>Grafico</string>
    </property>
    <addaction name="MenuSalvarGrafico"/>
    <addaction name="MenuAtivaLegenda"/>
    <addaction name="MenuAtivaLegendaGeral"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuGrafico"/>
  </widget>
  <action name="MenuCarregar">
   <property name="text">
    <string>Carregar Dados</string>
   </property>
  </action>
  <action name="MenuSalvarTabela">
   <property name="text">
    <string>Salvar Tabela</string>
   </property>
  </action>
  <action name="MenuSalvarGrafico">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Salvar Gráfico</string>
   </property>
  </action>
  <action name="MenuSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="MenuConfigUnit">
   <property name="text">
    <string>Configurar Unidades</string>
   </property>
  </action>
  <action name="MenuNovoMaterial">
   <property name="text">
    <string>Novo Material</string>
   </property>
  </action>
  <action name="MenuSalvarEquacoes">
   <property name="text">
    <string>Salvar Equações</string>
   </property>
  </action>
  <action name="MenuAtivaLegenda">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Legenda Material</string>
   </property>
  </action>
  <action name="MenuAtivaLegendaGeral">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Legenda Geral</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
